plugins/inputs/systemd_units/systemd_units_linux.go

```diff
@@ -320,6 +345,7 @@ func (s *SystemdUnits) Gather(acc telegraf.Accumulator) error {
 			"load":   state.LoadState,
 			"active": state.ActiveState,
 			"sub":    state.SubState,
+			"user":   s.user,
```

Comment:
Please only add the user tag for the "user" scope!

Reply:
sounds perfectly reasonable!